chesster.js

```diff
@@ -1071,8 +392,8 @@ controller.on('user_channel_join', function(bot, message) {
                         + "connected with the league. It is the easiest way for most "
                         + "of us to communicate and you will find that many of us "
                         + "are active in this community every day. Make yourself at home.");
-                convo.say(config.links.guide);
-                convo.say(config.links.rules);
+                convo.say(chesster.config["leagues"]["45+45"].links.guide);
```

Comment:
easy to throw this in a map all and hit all the leagues if it isnt too obnoxious with many leagues